walkthroughs/howto-mutual-tls-file-provided-by-acm/infrastructure/mesh.yaml

```diff
@@ -20,13 +20,13 @@ Parameters:
       - 1way-tls
```

Comment:
Should not include changes to this file?

Reply:
I think this might have been formatted by VSCode when i saved it, i will revert this change